Drivers/seds/TfLunaHwas.xml

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PackageFile xmlns="http://www.ccsds.org/schema/sois/seds" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.ccsds.org/schema/sois/seds seds.xsd">
+  <Package name="com/n7space/TfLunaHwas">
+    <DataTypeSet>
+      <EnumeratedDataType name="TfLunaHwas_ReadSM">
+        <LongDescription>Enum listing UART read data frame state machine states</LongDescription>
+        <IntegerDataEncoding encoding="unsigned" byteOrder="littleEndian" sizeInBits="8" />
+        <EnumerationList>
+          <Enumeration label="TfLunaHwas_ReadSM_NotFound" value="0" />
+          <Enumeration label="TfLunaHwas_ReadSM_Head" value="1"></Enumeration>
+          <Enumeration label="TfLunaHwas_ReadSM_DisL" value="3" />
+          <Enumeration label="TfLunaHwas_ReadSM_DisH" value="4" />
+          <Enumeration label="TfLunaHwas_ReadSM_StrengthL" value="5" />
+          <Enumeration label="TfLunaHwas_ReadSM_StrengthH" value="6" />
+          <Enumeration label="TfLunaHwas_ReadSM_TempL" value="7" />
+          <Enumeration label="TfLunaHwas_ReadSM_TempH" value="8" />
+          <Enumeration label="TfLunaHwas_ReadSM_Checksum" value="9" />
+        </EnumerationList>
+      </EnumeratedDataType>
+      <ContainerDataType name="TfLunaHwas_Data" shortDescription="Lidar data container (Distance, signal strength and temperature)">
+        <EntryList>
+          <Entry name="mDistance" type="com/n7space/Hwas/Word" />
+          <Entry name="mStrength" type="com/n7space/Hwas/Word" />
+          <Entry name="mTemp" type="com/n7space/Hwas/Word" />
+        </EntryList>
+      </ContainerDataType>
+      <EnumeratedDataType name="TfLunaHwas_DataFrameErr">
+        <LongDescription>Enum listing possible rx errors</LongDescription>
+        <IntegerDataEncoding encoding="unsigned" byteOrder="littleEndian" sizeInBits="8" />
+        <EnumerationList>
+          <Enumeration label="TfLunaHwas_DataFrameErr_NoHead1" value="0" />

```

Comment:
OK... So is this header, or just head (I would be suprised, but if this is in the Luna's docs....)?

Reply:
Yeah, ok. It should be called a header. Nevertheless, the documentation uses a word head, referring to the message header. (but not in the case of the first two bytes of that particular message, refer to page 24 of this manual https://download.kamami.pl/p579517-15978504093169389SJ-PM-TF-Luna%20A05%20Product%20Manual.pdf)

Reply:
OK, so if the docs say Head, let's stay with Head.